ansible/vars/softlayer/softlayer_public.yml

```diff
@@ -270,3 +270,5 @@ localsettings:
   USE_PARTITIONED_DATABASE: True
   WS4REDIS_CONNECTION_HOST: "{{ groups.redis.0 }}"
   XFORMS_PLAYER_URL: "http://{{ groups.touchforms.0 }}:4444/"
+  ENIKSHAY_PRIVATE_API_PASSWORD: {'BI': 'ppia-bi21', 'GU': 'ppia-gu21', 'MH': 'ppia-mh14'}
```

Comment:
PRIVATE is for PRIVATE API. There are two different integration, one private and one public.